eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java

```diff
@@ -40,7 +40,7 @@ public void save(String sagaId, Set<EnlistedAggregate> enlistedAggregates) {
                 return new EnlistedAggregate(ClassUtils.getClass(rs.getString("aggregate_type")), rs.getString("aggregate_id"));
               } catch (ClassNotFoundException e) {
                 logger.error("Class not found", e);
-                throw new RuntimeException();
+                throw new RuntimeException("Class not found", e);
```

Comment:
This message simply repeats the exception and so provides no useful information.
A more useful message would be one that describes *what* class isn't found, e.g. `RuntimeException("Aggregate type  not found", e)`

Reply:
ok, message for exception is "Aggregate type not found", should I use also logging here?

Reply:
Isn't that on line 42?

Reply:
Yes, should I keep it as is?